assets/boilerplate/style.css

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
This is is a lot to remove for any users who wants to create their own theme.
May I recommend limiting this to the alignments that prevents content from running up to the edges?

Reply:
I wonder if it would be better to ship this with no alignment code at all?

Reply:
It might be, because everyone would need to remove that once that spacing is solved upstream.